CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Change Log
+## 1.11.0 - 2019-03-22
+### Added
+* Add IoT Edge marketplace. User can view and create Edge modules from marketplace.
```

Comment:
from Azure Marketplace

Reply:
There so many Azure inside, do we really need this? :joy:
Add **Azure** IoT Edge marketplace. User can view and create **Azure** IoT Edge modules from **Azure** Marketplace

Reply:
Or just :
Add Azure IoT Edge marketplace. User can view and create Azure IoT Edge modules from it

Reply:
Upper case for "Marketplace"?

Reply:
How about `View and import Azure IoT Edge modules from Azure Marketplace`?